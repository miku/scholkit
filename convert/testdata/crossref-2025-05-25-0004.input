{
  "indexed": {
    "date-parts": [
      [
        2025,
        5,
        25
      ]
    ],
    "date-time": "2025-05-25T01:32:02Z",
    "timestamp": 1748136722713,
    "version": "3.40.5"
  },
  "reference-count": 37,
  "publisher": "American Society for Microbiology",
  "issue": "4",
  "license": [
    {
      "start": {
        "date-parts": [
          [
            2017,
            4,
            1
          ]
        ],
        "date-time": "2017-04-01T00:00:00Z",
        "timestamp": 1491004800000
      },
      "content-version": "tdm",
      "delay-in-days": 0,
      "URL": "https://journals.asm.org/non-commercial-tdm-license"
    }
  ],
  "funder": [
    {
      "name": "Thailand Research Fund (TRF) and Khon Kaen University, Thailand",
      "award": [
        "TRG5680032"
      ]
    },
    {
      "name": "Khon Kaen University (KKU) Research Fund, Thailand",
      "award": [
        "KKU590302"
      ]
    }
  ],
  "content-domain": {
    "domain": [
      "journals.asm.org"
    ],
    "crossmark-restriction": true
  },
  "short-container-title": [
    "Infect Immun"
  ],
  "published-print": {
    "date-parts": [
      [
        2017,
        4
      ]
    ]
  },
  "abstract": "<jats:title>ABSTRACT</jats:title>\n          <jats:p>\n            Persistent infection with\n            <jats:named-content content-type=\"genus-species\">Opisthorchis viverrini</jats:named-content>\n            causes hepatobiliary abnormalities, predisposing infected individuals to cholangiocarcinoma (CCA). In addition,\n            <jats:named-content content-type=\"genus-species\">Helicobacter pylori</jats:named-content>\n            is highly prevalent in most countries and is a possible risk factor for CCA; however, its role in enhancing hepatobiliary abnormality is unclear. Here, we investigated the effects of coinfection with\n            <jats:named-content content-type=\"genus-species\">H. pylori</jats:named-content>\n            and\n            <jats:named-content content-type=\"genus-species\">O. viverrini</jats:named-content>\n            on hepatobiliary abnormality. Hamsters were divided into four groups: (i) normal, (ii)\n            <jats:named-content content-type=\"genus-species\">H. pylori</jats:named-content>\n            infected (HP), (iii)\n            <jats:named-content content-type=\"genus-species\">O. viverrini</jats:named-content>\n            infected (OV), and (iv)\n            <jats:named-content content-type=\"genus-species\">O. viverrini</jats:named-content>\n            and\n            <jats:named-content content-type=\"genus-species\">H. pylori</jats:named-content>\n            infected (OV+HP). At 6 months postinfection, PCR and immunohistochemistry were used to test for the presence of\n            <jats:named-content content-type=\"genus-species\">H. pylori</jats:named-content>\n            in the stomach, gallbladder, and liver. In the liver,\n            <jats:named-content content-type=\"genus-species\">H. pylori</jats:named-content>\n            was detected in the following order: OV+HP, 5 of 8 (62.5%); HP, 2 of 5 (40%); OV, 2 of 8 (25%).\n            <jats:named-content content-type=\"genus-species\">H. pylori</jats:named-content>\n            was not detected in normal (control) liver tissues. Coinfection induced the most severe hepatobiliary abnormalities, including periductal fibrosis, cholangitis, and bile duct hyperplasia, leading to a significantly decreased survival rate of experimental animals. The greatest thickness of periductal fibrosis was associated with a significant increase in fibrogenesis markers (expression of alpha smooth muscle actin and transforming growth factor beta). Quantitative reverse transcription-PCR revealed that the highest expression levels of genes for proinflammatory cytokines (interleukin-1 [\n            <jats:italic>IL-1</jats:italic>\n            ],\n            <jats:italic>IL-6</jats:italic>\n            , and tumor necrosis factor alpha) were also observed in the OV+HP group. These results suggest that coinfection with\n            <jats:named-content content-type=\"genus-species\">H. pylori</jats:named-content>\n            and\n            <jats:named-content content-type=\"genus-species\">O. viverrini</jats:named-content>\n            increased the severity of hepatobiliary abnormalities to a greater extent than either single infection did.\n          </jats:p>",
  "DOI": "10.1128/iai.00009-17",
  "type": "journal-article",
  "created": {
    "date-parts": [
      [
        2017,
        1,
        31
      ]
    ],
    "date-time": "2017-01-31T02:20:26Z",
    "timestamp": 1485829226000
  },
  "update-policy": "https://doi.org/10.1128/asmj-crossmark-policy-page",
  "source": "Crossref",
  "is-referenced-by-count": 33,
  "title": [
    "Coinfection with Helicobacter pylori and Opisthorchis viverrini Enhances the Severity of Hepatobiliary Abnormalities in Hamsters"
  ],
  "prefix": "10.1128",
  "volume": "85",
  "author": [
    {
      "given": "Rungtiwa",
      "family": "Dangtakot",
      "sequence": "first",
      "affiliation": [
        {
          "name": "Science Program in Medical Technology, Faculty of Associated Medical Sciences, Khon Kaen University, Khon Kaen, Thailand"
        },
        {
          "name": "Center for Research and Development of Medical Diagnostic Laboratories, Faculty of Associated Medical Sciences, Khon Kaen University, Khon Kaen, Thailand"
        }
      ]
    },
    {
      "given": "Somchai",
      "family": "Pinlaor",
      "sequence": "additional",
      "affiliation": [
        {
          "name": "Department of Parasitology, Faculty of Medicine, Khon Kaen University, Khon Kaen, Thailand"
        },
        {
          "name": "Liver Fluke and Cholangiocarcinoma Research Center, Khon Kaen University, Khon Kaen, Thailand"
        }
      ]
    },
    {
      "given": "Upsornsawan",
      "family": "Itthitaetrakool",
      "sequence": "additional",
      "affiliation": [
        {
          "name": "Biomedical Science Program, Graduate School, Khon Kaen University, Khon Kaen, Thailand"
        }
      ]
    },
    {
      "given": "Apisit",
      "family": "Chaidee",
      "sequence": "additional",
      "affiliation": [
        {
          "name": "Department of Parasitology, Faculty of Medicine, Khon Kaen University, Khon Kaen, Thailand"
        },
        {
          "name": "Liver Fluke and Cholangiocarcinoma Research Center, Khon Kaen University, Khon Kaen, Thailand"
        }
      ]
    },
    {
      "given": "Chariya",
      "family": "Chomvarin",
      "sequence": "additional",
      "affiliation": [
        {
          "name": "Liver Fluke and Cholangiocarcinoma Research Center, Khon Kaen University, Khon Kaen, Thailand"
        },
        {
          "name": "Department of Microbiology, Faculty of Medicine, Khon Kaen University, Khon Kaen, Thailand"
        }
      ]
    },
    {
      "given": "Arunnee",
      "family": "Sangka",
      "sequence": "additional",
      "affiliation": [
        {
          "name": "Center for Research and Development of Medical Diagnostic Laboratories, Faculty of Associated Medical Sciences, Khon Kaen University, Khon Kaen, Thailand"
        }
      ]
    },
    {
      "given": "Chotechana",
      "family": "Wilailuckana",
      "sequence": "additional",
      "affiliation": [
        {
          "name": "Center for Research and Development of Medical Diagnostic Laboratories, Faculty of Associated Medical Sciences, Khon Kaen University, Khon Kaen, Thailand"
        }
      ]
    },
    {
      "ORCID": "https://orcid.org/0000-0001-7436-3355",
      "authenticated-orcid": false,
      "given": "Porntip",
      "family": "Pinlaor",
      "sequence": "additional",
      "affiliation": [
        {
          "name": "Center for Research and Development of Medical Diagnostic Laboratories, Faculty of Associated Medical Sciences, Khon Kaen University, Khon Kaen, Thailand"
        },
        {
          "name": "Liver Fluke and Cholangiocarcinoma Research Center, Khon Kaen University, Khon Kaen, Thailand"
        }
      ]
    }
  ],
  "member": "235",
  "reference": [
    {
      "key": "e_1_3_3_2_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1002/jhbp.62"
    },
    {
      "key": "e_1_3_3_3_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1016/j.pt.2012.07.006"
    },
    {
      "key": "e_1_3_3_4_2",
      "first-page": "1",
      "article-title": "Biological agents. Volume 100 B. A review of human carcinogens",
      "volume": "100",
      "author": "IARC Working Group on the Evaluation of Carcinogenic Risks to Humans",
      "year": "2012",
      "unstructured": "IARC Working Group on the Evaluation of Carcinogenic Risks to Humans. 2012. Biological agents. Volume 100 B. A review of human carcinogens. IARC Monogr Eval Carcinog Risks Hum100(Pt B):1â€“441.",
      "journal-title": "IARC Monogr Eval Carcinog Risks Hum"
    },
    {
      "key": "e_1_3_3_5_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1111/ijcp.12264"
    },
    {
      "key": "e_1_3_3_6_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1038/nrgastro.2013.243"
    },
    {
      "key": "e_1_3_3_7_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1111/hel.12077"
    },
    {
      "key": "e_1_3_3_8_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1371/journal.pone.0070265"
    },
    {
      "key": "e_1_3_3_9_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1371/journal.pone.0136775"
    },
    {
      "key": "e_1_3_3_10_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.5009/gnl.2010.4.1.60"
    },
    {
      "key": "e_1_3_3_11_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1111/j.1477-2574.2011.00423.x"
    },
    {
      "key": "e_1_3_3_12_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.7314/APJCP.2015.16.5.1751"
    },
    {
      "key": "e_1_3_3_13_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1016/j.niox.2004.08.004"
    },
    {
      "key": "e_1_3_3_14_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1016/j.ijpara.2008.12.002"
    },
    {
      "key": "e_1_3_3_15_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1002/ijc.25266"
    },
    {
      "key": "e_1_3_3_16_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1371/journal.pone.0093943"
    },
    {
      "key": "e_1_3_3_17_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1002/hep.23134"
    },
    {
      "key": "e_1_3_3_18_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1111/hel.12298"
    },
    {
      "key": "e_1_3_3_19_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1038/nm1663"
    },
    {
      "key": "e_1_3_3_20_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1038/labinvest.2009.90"
    },
    {
      "key": "e_1_3_3_21_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1016/j.biopha.2013.03.004"
    },
    {
      "key": "e_1_3_3_22_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1128/JCM.00879-09"
    },
    {
      "key": "e_1_3_3_23_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1128/JCM.38.10.3722-3728.2000"
    },
    {
      "key": "e_1_3_3_24_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1128/JCM.38.5.1811-1817.2000"
    },
    {
      "key": "e_1_3_3_25_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1016/S0147-9571(98)00018-6"
    },
    {
      "key": "e_1_3_3_26_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1111/hel.12265"
    },
    {
      "key": "e_1_3_3_27_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1111/j.1349-7006.2002.tb01327.x"
    },
    {
      "key": "e_1_3_3_28_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1111/j.1469-0691.2009.02719.x"
    },
    {
      "key": "e_1_3_3_29_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1371/journal.pone.0165798"
    },
    {
      "key": "e_1_3_3_30_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1111/j.1600-0463.2006.apm_425.x"
    },
    {
      "key": "e_1_3_3_31_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1007/s00534-006-1152-y"
    },
    {
      "key": "e_1_3_3_32_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1007/s12664-012-0178-0"
    },
    {
      "key": "e_1_3_3_33_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1111/j.1477-2574.2012.00533.x"
    },
    {
      "key": "e_1_3_3_34_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1016/S0016-5085(98)70365-8"
    },
    {
      "key": "e_1_3_3_35_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1128/IAI.01788-06"
    },
    {
      "key": "e_1_3_3_36_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1093/carcin/24.1.133"
    },
    {
      "key": "e_1_3_3_37_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1016/j.jhep.2007.07.006"
    },
    {
      "key": "e_1_3_3_38_2",
      "doi-asserted-by": "publisher",
      "DOI": "10.1006/meth.2001.1262"
    }
  ],
  "container-title": [
    "Infection and Immunity"
  ],
  "language": "en",
  "link": [
    {
      "URL": "https://journals.asm.org/doi/pdf/10.1128/IAI.00009-17",
      "content-type": "application/pdf",
      "content-version": "vor",
      "intended-application": "text-mining"
    },
    {
      "URL": "https://journals.asm.org/doi/pdf/10.1128/IAI.00009-17",
      "content-type": "unspecified",
      "content-version": "vor",
      "intended-application": "similarity-checking"
    }
  ],
  "deposited": {
    "date-parts": [
      [
        2022,
        3,
        5
      ]
    ],
    "date-time": "2022-03-05T10:37:24Z",
    "timestamp": 1646476644000
  },
  "score": 0.0,
  "resource": {
    "primary": {
      "URL": "https://journals.asm.org/doi/10.1128/IAI.00009-17"
    }
  },
  "editor": [
    {
      "given": "Steven R.",
      "family": "Blanke",
      "sequence": "additional",
      "affiliation": []
    }
  ],
  "issued": {
    "date-parts": [
      [
        2017,
        4
      ]
    ]
  },
  "references-count": 37,
  "journal-issue": {
    "issue": "4",
    "published-print": {
      "date-parts": [
        [
          2017,
          4
        ]
      ]
    }
  },
  "alternative-id": [
    "10.1128/IAI.00009-17"
  ],
  "URL": "https://doi.org/10.1128/iai.00009-17",
  "ISSN": [
    "0019-9567",
    "1098-5522"
  ],
  "issn-type": [
    {
      "type": "print",
      "value": "0019-9567"
    },
    {
      "type": "electronic",
      "value": "1098-5522"
    }
  ],
  "published": {
    "date-parts": [
      [
        2017,
        4
      ]
    ]
  },
  "assertion": [
    {
      "value": "2017-01-05",
      "order": 0,
      "name": "received",
      "label": "Received",
      "group": {
        "name": "publication_history",
        "label": "Publication History"
      }
    },
    {
      "value": "2017-01-22",
      "order": 1,
      "name": "accepted",
      "label": "Accepted",
      "group": {
        "name": "publication_history",
        "label": "Publication History"
      }
    },
    {
      "value": "2017-03-23",
      "order": 2,
      "name": "published",
      "label": "Published",
      "group": {
        "name": "publication_history",
        "label": "Publication History"
      }
    }
  ]
}
