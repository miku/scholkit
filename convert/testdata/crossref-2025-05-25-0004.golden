{
    "source": "crossref",
    "contribs": [
        {
            "extra": {
                "more_affiliation": [
                    "Center for Research and Development of Medical Diagnostic Laboratories, Faculty of Associated Medical Sciences, Khon Kaen University, Khon Kaen, Thailand"
                ]
            },
            "given_name": "Rungtiwa",
            "raw_name": "Rungtiwa Dangtakot",
            "role": "author",
            "surname": "Dangtakot",
            "raw_affiliation": "Science Program in Medical Technology, Faculty of Associated Medical Sciences, Khon Kaen University, Khon Kaen, Thailand"
        },
        {
            "extra": {
                "more_affiliation": [
                    "Liver Fluke and Cholangiocarcinoma Research Center, Khon Kaen University, Khon Kaen, Thailand"
                ]
            },
            "given_name": "Somchai",
            "raw_name": "Somchai Pinlaor",
            "role": "author",
            "surname": "Pinlaor",
            "raw_affiliation": "Department of Parasitology, Faculty of Medicine, Khon Kaen University, Khon Kaen, Thailand"
        },
        {
            "extra": {},
            "given_name": "Upsornsawan",
            "raw_name": "Upsornsawan Itthitaetrakool",
            "role": "author",
            "surname": "Itthitaetrakool",
            "raw_affiliation": "Biomedical Science Program, Graduate School, Khon Kaen University, Khon Kaen, Thailand"
        },
        {
            "extra": {
                "more_affiliation": [
                    "Liver Fluke and Cholangiocarcinoma Research Center, Khon Kaen University, Khon Kaen, Thailand"
                ]
            },
            "given_name": "Apisit",
            "raw_name": "Apisit Chaidee",
            "role": "author",
            "surname": "Chaidee",
            "raw_affiliation": "Department of Parasitology, Faculty of Medicine, Khon Kaen University, Khon Kaen, Thailand"
        },
        {
            "extra": {
                "more_affiliation": [
                    "Department of Microbiology, Faculty of Medicine, Khon Kaen University, Khon Kaen, Thailand"
                ]
            },
            "given_name": "Chariya",
            "raw_name": "Chariya Chomvarin",
            "role": "author",
            "surname": "Chomvarin",
            "raw_affiliation": "Liver Fluke and Cholangiocarcinoma Research Center, Khon Kaen University, Khon Kaen, Thailand"
        },
        {
            "extra": {},
            "given_name": "Arunnee",
            "raw_name": "Arunnee Sangka",
            "role": "author",
            "surname": "Sangka",
            "raw_affiliation": "Center for Research and Development of Medical Diagnostic Laboratories, Faculty of Associated Medical Sciences, Khon Kaen University, Khon Kaen, Thailand"
        },
        {
            "extra": {},
            "given_name": "Chotechana",
            "raw_name": "Chotechana Wilailuckana",
            "role": "author",
            "surname": "Wilailuckana",
            "raw_affiliation": "Center for Research and Development of Medical Diagnostic Laboratories, Faculty of Associated Medical Sciences, Khon Kaen University, Khon Kaen, Thailand"
        },
        {
            "extra": {
                "more_affiliation": [
                    "Liver Fluke and Cholangiocarcinoma Research Center, Khon Kaen University, Khon Kaen, Thailand"
                ]
            },
            "given_name": "Porntip",
            "raw_name": "Porntip Pinlaor",
            "role": "author",
            "surname": "Pinlaor",
            "raw_affiliation": "Center for Research and Development of Medical Diagnostic Laboratories, Faculty of Associated Medical Sciences, Khon Kaen University, Khon Kaen, Thailand"
        },
        {
            "extra": {},
            "given_name": "Steven R.",
            "raw_name": "Steven R. Blanke",
            "role": "editor",
            "surname": "Blanke"
        }
    ],
    "ext_ids": {
        "doi": "10.1128/iai.00009-17"
    },
    "extra": {
        "arxiv": {},
        "dblp": {},
        "openalex": {
            "open_access": {}
        },
        "crossref": {},
        "oai": {}
    },
    "id": "crossref-54c8d3e8345f94c5976b7d55aa4f6a76e8092ed2",
    "release_date": "49054-01-20",
    "release_type": "article-journal",
    "title": "Coinfection with Helicobacter pylori and Opisthorchis viverrini Enhances the Severity of Hepatobiliary Abnormalities in Hamsters"
}